test/Altinn.AccessManagement.Tests/Controllers/HomeControllerTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Altinn.AccessManagement.Controllers;
+using Altinn.AccessManagement.Core.Clients;
+using Altinn.AccessManagement.Tests.Mocks;
+using Altinn.AccessManagement.Tests.Util;
+using Altinn.AccessManagement.Tests.Utils;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using static Microsoft.ApplicationInsights.MetricDimensionNames.TelemetryContext;
+
+namespace Altinn.AccessManagement.Tests.Controllers
+{
+    /// <summary>
+    /// Test class for <see cref="HomeController"></see>
+    /// </summary>
+    [Collection("HomeController Tests")]
+    public class HomeControllerTest : IClassFixture<CustomWebApplicationFactory<HomeController>>
```

Comment:
Should we have a test with cookie also? I belive we have som in apps. 

Just to make sure both works?